Tools/CISupport/ews-build/steps.py

```diff
@@ -113,6 +113,12 @@ class GitHub(object):
     REQUEST_MERGE_QUEUE_LABEL = 'request-merge-queue'
     BLOCKED_LABEL = 'merging-blocked'
     SKIP_EWS_LABEL = 'skip-ews'
+    NO_FAILURE_LIMITS = 'no-failure-limits'
```

Comment:
Nit: `NO_FAILURE_LIMITS` => `NO_FAILURE_LIMITS_LABEL` 
(for consistency with other similar variables)